runWebpack.sh

```diff
@@ -78,6 +78,7 @@ gitops_config() {
   export HOSTNAME="$hostname"
   export APP="$portalApp"
   if [[ "$portalApp" == "gitops" ]]; then
+    gitopsFolder="../$gitRepo/$hostname"
```

Comment:
is this used anywhere? just saw its the only change in the file

Reply:
not used. fixed in 6723c9f